executor/src/witgen/multiplicity_column_generator.rs

```diff
@@ -38,20 +35,10 @@ impl<'a, T: FieldElement> MultiplicityColumnGenerator<'a, T> {
     ) -> HashMap<String, Vec<T>> {
```

Comment:
Documentation comment above: Is the term "phantom lookups" still correct?

Reply:
Fixed!